### ISSUE TITLE
Legend of Question Types in Quiz (UI)

### ISSUE THREAD

**Issue by ankitjavalkar** [FOSSEE/online_test#525]
Implementation of the legend in the running Quiz seems to be slightly unpolished in the new UI. May be this could be made better.

**Comment by prathamesh920** [comment]
Yes I agree! Also, it shows the question ids instead of the question numbers!
The id number is completely irrelevant when compared with the navigator question number.
